.github/workflows/publish-commit.yml

```diff
@@ -18,8 +18,46 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - if: github.event.issue.pull_request
```

Comment:
I wonder if we can check the role in the previous step? using `github.event.comment.author_association == 'MEMBER'` ?

Reply:
Perhaps that works too, but I won't able to test it now 😅